crypto/rsa_extra/rsa_test.cc

```diff
@@ -578,13 +578,71 @@ TEST(RSATest, OnlyDGiven) {
   ASSERT_TRUE(BN_hex2bn(&key2->d, kD));
   key2->flags |= RSA_FLAG_NO_BLINDING;
 
+  // While keys defined only in terms of |n| and |d| must be functional, our
+  // validation logic doesn't consider them "valid".
+  EXPECT_FALSE(RSA_check_key(key2.get()));
+
   ASSERT_LE(RSA_size(key2.get()), sizeof(buf));
   EXPECT_TRUE(RSA_sign(NID_sha256, kDummyHash, sizeof(kDummyHash), buf,
                        &buf_len, key2.get()));
 
   // Verify the signature with |key|. |key2| has no public exponent.
   EXPECT_TRUE(RSA_verify(NID_sha256, kDummyHash, sizeof(kDummyHash), buf,
                          buf_len, key.get()));
+
+  // Perform the same test case as above ("ACCP-style" key specified in terms
```

Comment:
```suggestion
  // Perform the same test case as above ("JCA-style" key specified in terms
```

Reply:
i have no issue with that. perhaps we can consolidate discussion with @SalusaSecondus around the naming issue on [this thread](https://github.com/awslabs/aws-lc/pull/436#discussion_r850827317)?